src/Sylius/Bundle/CoreBundle/spec/Form/Type/Checkout/PaymentStepTypeSpec.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Bundle\CoreBundle\Form\Type\Checkout;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Sylius\Bundle\ResourceBundle\Form\Type\AbstractResourceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class PaymentStepTypeSpec extends ObjectBehavior
+{
+    function let()
+    {
+        $this->beConstructedWith(['sylius', 'checkout']);
+    }
+    
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Bundle\CoreBundle\Form\Type\Checkout\PaymentStepType');
+    }
+
+    function it_is_an_abstract_type()
+    {
+        $this->shouldHaveType(AbstractResourceType::class);
+    }
+
+    function it_builds_form_with_proper_fields(FormBuilderInterface $builder)
+    {
+        $builder
+            ->add('payments', Argument::type('string'), [
```

Comment:
Is it really necessary to use ``Argument::...``? Can't it be just ``'collection'``?